editor/plugins/shader_editor_plugin.cpp

```diff
@@ -649,6 +694,80 @@ void ShaderEditor::_make_context_menu(bool p_selection, Vector2 p_position) {
 	context_menu->popup();
 }
 
+void ShaderEditor::_tree_activate_shader()
+{
+	TreeItem *selected = shader_dependency_tree->get_selected();
+	if (selected) {
+		String path = selected->get_metadata(0);
+
+		auto rollingCode = shader_rolling_code.find(path);
```

Comment:
Don't use `auto` - it's forbidden by Juan.

Reply:
And identifiers should use `snake_case`.